src/applications/personalization/profile/mocks/personal-information/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable camelcase */
```

Comment:
Moved this mock into its own folder to adhere to the other mock's organizational structure